### PR TITLE
Copy kinto.ini from sample

### PR DESCRIPTION
**Pull request by leplatrem**
Just trying something... Thoughts ?

### CODE REVIEW

File: kinto/__main__.py [hunk 1]
```diff
@@ -34,6 +37,10 @@ def main(args=None):
 
     args = vars(parser.parse_args())
     config_file = args['ini_file']
+
+    if not os.path.exists(config_file):
+        shutil.copy('config/kinto.ini.sample', config_file)
```
Comment (Natim) [inline kinto/__main__.py]: :+1:

Comment (Natim) [inline kinto/__main__.py]: Shall we do that in the init command instead? and tell the person that she need to start ``kinto init`` first?

In the make file we can also add the CONFIG_FILE as a dependency and run ``kinto init`` automatically if the file doesn't exists.

File: docs/tutorials/run-kinto.rst [hunk 2]
```diff
@@ -73,8 +73,8 @@ Then install the package using the default configuration:
 ::
 
     pip install kinto
-    wget https://raw.githubusercontent.com/Kinto/kinto/master/config/kinto.ini
-    pserve kinto.ini
+    wget https://raw.githubusercontent.com/Kinto/kinto/master/config/kinto.ini.sample -O kinto.ini
+    kinto --ini kinto.ini start
```
Comment (leplatrem) [inline docs/tutorials/run-kinto.rst]: We could just put the sample in the egg and suggest to run `kinto start` only :)

File: kinto/__main__.py (hunk 1 above)
Comment (leplatrem) [inline kinto/__main__.py]: good idead!

File: docs/tutorials/run-kinto.rst (hunk 2 above)
Comment (Natim) [inline docs/tutorials/run-kinto.rst]: I would add the sample in the egg anyway + @oak11 work on the init command to do something like:

    kinto init
    kinto start

**Comment by leplatrem** [comment]
This was implemented using template in #278

**Pull request closed without merging**